website/docs/r/secretsmanager_secret.html.markdown

```diff
@@ -56,7 +56,7 @@ The following arguments are supported:
 
 ### replica
 
-* `kms_key_id` - (Optional) ARN, Key ID, or Alias.
+* `kms_key_id` - (Optional) ARN, Key ID  or Alias of the AWS KMS customer master key (CMK) within the region secret is replicated to. If one is not specified, then Secrets Manager defaults to using the AWS account's default managed key (`aws/secretsmanager`) in the region or creates one for use if non-existent.
```

Comment:
Minor suggestions here:
- Added the Oxford comma; this is totally stylistic but reads a bit better to me personally.
- Removed `customer master key (CMK)`. I'm not 100% on this change, and am willing to discard it. I'm basing this partially on the lack of mentioning CMK in the [API documentation](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_ReplicaRegionType.html) for `CreateSecret` and partial on an [AWS document](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#kms_keys) that I stumbled on while trying to confirm my thoughts. It has the following note: 

> AWS KMS is replacing the term customer master key (CMK) with AWS KMS key and KMS key. The concept has not changed. To prevent breaking changes, AWS KMS is keeping some variations of this term.

```suggestion
* `kms_key_id` - (Optional) ARN, Key ID, or Alias of the AWS KMS within the region secret is replicated to. If one is not specified, then Secrets Manager defaults to using the AWS account's default managed key (`aws/secretsmanager`) in the region or creates one for use if non-existent.
```

Reply:
Thanks for the review @justinretzolk . I knew about the naming change for KMS but for consistency of the documentation kept the `customer master key` reference to match the prior `kms_key_id` description.  I can fold a change for that into this one PR as well.